devito/ir/support/symregistry.py

```diff
@@ -30,7 +30,7 @@ def __init__(self):
         # passes, to maximize symbol (especially Dimension) reuse
         self.caches = {}
 
-    def make_name(self, prefix=None):
+    def make_name(self, prefix=None, increment_first=True):
```

Comment:
Any reason to not make the default case to keep first name

Reply:
I guess you would then have

float r = ...
float r1 = ...

and `r` is such a common name

Reply:
>I guess you would then have
>
>float r = ...
>float r1 = ...
>
>and r is such a common name
Correct. This behaviour is only desirable in some cases, hence the switch.